### PR TITLE
change WebDAV to iCal wording, fix #233

### LINKED ISSUE #233: Phrasing "Settings" on public page is confusing
Hi,

### Steps to reproduce
1. Add a calendar, share it publicly.
2. On the public URL of the calendar, try to access to the matching ICS file, to feed it into some external tool such as Google Agenda.

### Expected behaviour

There is no way to directly access (easily) the URL of the ICS file. I would have expected to have something like an input field with the URL of the ICS file somewhere, besides the "Download" button.

### Actual behaviour

The only solution I could find is to actually download the file, and rely on my browser to get the source URL of the ICS file, which is highly unpractical, and especially tricky for non-tech users.

### Server configuration

Latest version of NextCloud and calendar app as of today. The rest does not seem to be relevant for this issue.

Thanks!

### PR DESCRIPTION
**Pull request by jancborchardt**
Fix https://github.com/nextcloud/calendar/issues/233, please review @nextcloud/calendar

### CODE REVIEW

**Comment by jancborchardt** [comment]
@georgehrke if you prefer »iCal feed« then please update this and merge it. :) But the current state is not acceptable

**Comment by jancborchardt** [comment]
@nextcloud/calendar please review

@georgehrke I changed it to »iCal feed«. If anything else needs to be changed please just do it. We should merge it soon because the current thing is just wrong.

**Comment by georgehrke** [comment]
calendar - nextcloud chromium today at 12 39 03 pm

We are calling it Webcal within the calendar. Do you want me to rename it there?

**Comment by jancborchardt** [comment]
Yes, so it’s consistent. To »iCal link«, right?

**Comment by skjnldsv** [comment]
iCal seems like an apple product :D 

I won't mind WebCal or iCal imho. But without the 'url'.

**Comment by georgehrke** [comment]
> iCal seems like an apple product :D

The entire standard is called iCalendar :P

---
Will change the wording to »iCal link«

**Comment by eppfel** [comment]
If it is a url, it should be called url. Link is semantically incorrect. The only thing other than URL I can think of is address.

But not a big deal, just my 2 cents.

**Comment by skjnldsv** [comment]
To be faire I was thinking without url or link.
Just "WebCal" ^^'

File: templates/part.publicinformations.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ class="settings-button"
 					<div class="davbuttons">
 						<div class="btn-group">
 							<button class="button first" ng-model="$parent.publicdav" uib-btn-radio="'CalDAV'">CalDAV</button>
-							<button class="button last" ng-model="$parent.publicdav" uib-btn-radio="'WebDAV'">WebDAV</button>
+							<button class="button last" ng-model="$parent.publicdav" uib-btn-radio="'iCal feed'">iCal feed</button>
```
Comment (jancborchardt) [inline templates/part.publicinformations.php]: @georgehrke shouldn't we call it"iCal link" here too then?

Comment (georgehrke) [inline templates/part.publicinformations.php]: Hm, maybe.

**Pull request merged**